llvm/tools/repo2obj/R2OELFSymbolTable.h

```diff
@@ -131,9 +131,13 @@ template <typename ELFT> class SymbolTable {
 template <typename ELFT>
 unsigned SymbolTable<ELFT>::linkageToELFBinding(pstore::repo::linkage_type L) {
   switch (L) {
+  case pstore::repo::linkage_type::internal_no_symbol:
```

Comment:
Do we need logic to exclude `internal_no_symbol` from the ELF symbol table?

Reply:
The same as the weak linkage,  I tried to keep the same behavior as before. I will remove internal_no_symbol from the ELF symbol and test using the LLVM project build.

Reply:
> 
> 
> Do we need logic to exclude `internal_no_symbol` from the ELF symbol table?

I checked the ELF code, I don't think we need logic exclude `internal_no_symbol` from here.

In the GlobalValue.h file, the function `isLocalLinkage` defined that the private and internal linkages are the local linkage. The symbol will be set to the ELF::STB_LOCAL binding if the corresponding GV has a either private or internal linkage type.